django/db/backends/sqlite3/schema.py

```diff
@@ -11,26 +11,20 @@
 class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
 
     sql_delete_table = "DROP TABLE %(table)s"
-    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s)"
+    sql_create_fk = None
+    sql_create_inline_fk = "REFERENCES %(to_table)s (%(to_column)s) DEFERRABLE INITIALLY DEFERRED"
```

Comment:
I wonder how this will play with existing SQLite tables that don't have `DEFERRABLE INITIALLY DEFERRED` constraints. At this point I think the best course of action would be to mention it in the release notes.